### PR TITLE
[ATL-590] As an API User, I can send a quotation

### PR DESCRIPTION
**Pull request by mattkomarnicki**
### Link to issue 

https://teamleader.atlassian.net/browse/ATL-590

### Description

Go to: https://github.com/teamleadercrm/core/pull/29015

### CODE REVIEW

**Review by joaoinacio — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -2430,6 +2430,57 @@ Create a quotation.
...
+                       + type (enum, required)
+                           + Members
+                               + company
+                               + contact
+                       + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+                   + email_address: `info@teamleader.eu` (string, required)
+        + subject: `An offer you cannot refuse` (string, required)
+        + content: `Sign your offer here #LINK` (string, required) - the shortcode '#LINK' will be replaced with the cloudsign url
+        + attachments: `e76d381a-f7ea-04d1-9a32-14aab27bc1e2` (array[string], optional) - an array of file ids
+        + language: `en` (string, required)
```
Comment (Jdww) [inline apiary.apib]: Q/PP: Should this be an enum? Listening all available language codes

File: apiary.apib [hunk 2]
```diff
@@ -2430,6 +2430,57 @@ Create a quotation.
...
+        + from (object, optional)
+            + sender (object, required)
+                + type (enum, required)
+                    + Members
+                        + user
+                        + department
+                + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+            + email_address: `info@teamleader.eu` (string, required)
+        + recipients (object, required)
+            + to (array)
```
Comment (tiagobastos) [inline apiary.apib]: Q: Shouldn't this be marked as `required`?

**Review by tiagobastos — COMMENTED** [review]
We are missing an entry on the changelog.

File: apiary.apib (hunk 1 above)
Comment (mattkomarnicki) [reply apiary.apib]: Fixed: https://github.com/teamleadercrm/api/pull/654/commits/6fa31be6d03492cf4ffa897d91828321cbd49348

File: apiary.apib (hunk 2 above)
Comment (mattkomarnicki) [reply apiary.apib]: You're right @tiagobastos 

Fixed: https://github.com/teamleadercrm/api/pull/654/commits/0c74650f91449b389e55c1acfbf1b0620922af4d

**Comment by mattkomarnicki** [comment]
Altered the changelog: https://github.com/teamleadercrm/api/pull/654/commits/74416f468dfc9a03305383d012f7f2c3cd3b791e @tiagobastos

File: apiary.apib (hunk 1 above)
Comment (Jdww) [reply apiary.apib]: PP: Use a datastructure for the language enum values

Comment (mattkomarnicki) [reply apiary.apib]: Makes absolute sense: https://github.com/teamleadercrm/api/pull/654/commits/802b9fb27382e35203f7f4f8cc9849fa038494d9

**Review by Jdww — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

**Pull request merged**